.github/workflows/ci.yml

```diff
@@ -8,43 +8,98 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
+
+  test:
+
     runs-on: ubuntu-20.04
     continue-on-error: ${{ matrix.experimental }}
 
     strategy:
       matrix:
         include:
           - mediawiki_version: '1.35'
-            semantic_mediawiki_version: 4.0.2
+            smw_version: '4.1.0'
+            php_version: 7.4
+            dt_version: 3.1
+            ps_version: 0.6.1
+            database_type: mysql
+            database_image: "mysql:5.7"
             coverage: true
             experimental: false
           - mediawiki_version: '1.35'
-            semantic_mediawiki_version: dev-master
+            smw_version: '4.1.2'
+            php_version: 7.4
+            dt_version: 3.1
+            ps_version: 0.6.1
+            database_type: sqlite
+            database_image: "mysql:5.7"
```

Comment:
@gesinn-it-evl this looks strange: having sqlite with an mysql image. Is this something that could be changed? Even if it does not matter from the technical point of view, it might confuse others.